### ISSUE TITLE
git error 128 when I run `npm init video`

### ISSUE THREAD

**Issue by TotoShampoin** [remotion-dev/remotion#1720]
When I run `npm init video`, I get this error:
```
totoshampoin@DellG3-LM21-Toto:~/Documents/cours/semestre-1/audiovisuel/court-metrage-subtitle$ npm init video
Error: Command failed with exit code 128: git rev-parse --show-toplevel
fatal: Not a git repository (or any of the parent directories): .git
    at makeError (/home/totoshampoin/.npm/_npx/a2780e854c8aa659/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/totoshampoin/.npm/_npx/a2780e854c8aa659/node_modules/execa/index.js:118:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async checkGitAvailability (/home/totoshampoin/.npm/_npx/a2780e854c8aa659/node_modules/create-video/dist/init.js:20:24)
    at async init (/home/totoshampoin/.npm/_npx/a2780e854c8aa659/node_modules/create-video/dist/init.js:62:20) {
  shortMessage: 'Command failed with exit code 128: git rev-parse --show-toplevel',
  command: 'git rev-parse --show-toplevel',
  escapedCommand: 'git rev-parse --show-toplevel',
  exitCode: 128,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: "fatal: Not a git repository (or any of the parent directories): .git",
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm ERR! code 1
npm ERR! path /home/totoshampoin/Documents/cours/semestre-1/audiovisuel/court-metrage-subtitle
npm ERR! command failed
npm ERR! command sh -c create-video

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/totoshampoin/.npm/_logs/2023-01-21T16_24_38_571Z-debug-0.log
```

I'm using a recent install of Linux Mint 21, on a Dell G3 15
npm version: 9.3.1
node version: 18.13.0

I have git 2.34.1 installed, but not linked to github.

What might be the issue?

**Comment by JonnyBurger** [comment]
Thanks for reporting this! We released this code very recently (preventing creating a new video inside a Git repo that already exists) and looks like we made a bugfix.

Will release it now, you can create a new video using `npx create-video@latest`!

**Comment by TotoShampoin** [comment]
I tried again, got the same error

This is the output when I run it in verbose mode
```
totoshampoin@DellG3-LM21-Toto:~/Documents/cours/semestre-1/audiovisuel/subtitles$ npx -verbose create-video@latest
npm verb cli /usr/bin/node /usr/lib/node_modules/npm/bin/npm-cli.js
npm info using npm@9.3.1
npm info using node@v18.13.0
npm verb title npm exec create-video@latest
npm verb argv "exec" "--loglevel" "verbose" "--" "create-video@latest"
npm verb logfile logs-max:10 dir:/home/totoshampoin/.npm/_logs/2023-01-22T16_15_24_903Z-
npm verb logfile /home/totoshampoin/.npm/_logs/2023-01-22T16_15_24_903Z-debug-0.log
npm http fetch GET 200 https://registry.npmjs.org/create-video 826ms (cache revalidated)
Error: Command failed with exit code 128: git rev-parse --show-toplevel
fatal: ni ceci ni aucun de ses répertoires parents n'est un dépôt git : .git
    at makeError (/home/totoshampoin/.npm/_npx/f45e76a8b7cfddff/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/totoshampoin/.npm/_npx/f45e76a8b7cfddff/node_modules/execa/index.js:118:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async checkGitAvailability (/home/totoshampoin/.npm/_npx/f45e76a8b7cfddff/node_modules/create-video/dist/init.js:20:24)
    at async init (/home/totoshampoin/.npm/_npx/f45e76a8b7cfddff/node_modules/create-video/dist/init.js:62:20) {
  shortMessage: 'Command failed with exit code 128: git rev-parse --show-toplevel',
  command: 'git rev-parse --show-toplevel',
  escapedCommand: 'git rev-parse --show-toplevel',
  exitCode: 128,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: "fatal: ni ceci ni aucun de ses répertoires parents n'est un dépôt git : .git",
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
npm verb exit 1
npm verb code 1
```
(It's the same error message, but I didn't translate it from French this time)

And here is the log file:
```
6_15_24_903Z-debug-0.log
0 verbose cli /usr/bin/node /usr/lib/node_modules/npm/bin/npm-cli.js
1 info using npm@9.3.1
2 info using node@v18.13.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 2ms
5 timing config:load:file:/usr/lib/node_modules/npm/npmrc Completed in 0ms
6 timing config:load:builtin Completed in 0ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/home/totoshampoin/Documents/cours/semestre-1/audiovisuel/subtitles/.npmrc Completed in 0ms
10 timing config:load:project Completed in 3ms
11 timing config:load:file:/home/totoshampoin/.npmrc Completed in 1ms
12 timing config:load:user Completed in 1ms
13 timing config:load:file:/usr/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 1ms
16 timing config:load Completed in 9ms
17 timing npm:load:configload Completed in 9ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 0ms
20 verbose title npm exec create-video@latest
21 verbose argv "exec" "--loglevel" "verbose" "--" "create-video@latest"
22 timing npm:load:setTitle Completed in 1ms
23 timing config:load:flatten Completed in 2ms
24 timing npm:load:display Completed in 4ms
25 verbose logfile logs-max:10 dir:/home/totoshampoin/.npm/_logs/2023-01-22T16_15_24_903Z-
26 verbose logfile /home/totoshampoin/.npm/_logs/2023-01-22T16_15_24_903Z-debug-0.log
27 timing npm:load:logFile Completed in 4ms
28 timing npm:load:timers Completed in 0ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 19ms
31 silly logfile start cleaning logs, removing 1 files
32 silly logfile done cleaning log files
33 timing arborist:ctor Completed in 1ms
34 http fetch GET 200 https://registry.npmjs.org/create-video 826ms (cache revalidated)
35 timing arborist:ctor Completed in 0ms
36 timing arborist:ctor Completed in 0ms
37 timing arborist:ctor Completed in 0ms
38 timing command:exec Completed in 1099ms
39 verbose exit 1
40 timing npm Completed in 1127ms
41 verbose code 1
```

**Comment by JonnyBurger** [comment]
@TotoShampoin Sorry about that!
Putting in another fix in #1724

**Comment by TotoShampoin** [comment]
It works now! Thank you
